### PR TITLE
fix: explicitly disable --short in scheduler-perf jobs

### PR DESCRIPTION
**Pull request by sanposhiho**
explicitly disable --short in scheduler-perf jobs otherwise the default is `--short=true`, which will run a fewer tests after https://github.com/kubernetes/kubernetes/pull/127269.

/cc @macsko

### CODE REVIEW

**Comment by macsko** [comment]
What about `pull-scheduler-perf` presubmit in `config/jobs/kubernetes/sig-scalability/sig-scalability-presubmit-jobs.yaml`?

**Comment by sanposhiho** [comment]
Oh, I didn't know that one existed. Done. Also, synced the change of `KUBE_KEEP_VERBOSE_TEST_OUTPUT` too, which is added in https://github.com/kubernetes/test-infra/pull/33447.

**Comment by sanposhiho** [comment]
/cc @mm4tt @wojtek-t 

for approval

File: config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml [hunk 1]
```diff
@@ -755,6 +757,8 @@ periodics:
       # even on very powerful machines
       - name: BENCHTIME
         value: 1ns
+      - name: SHORT
```
Comment (krzyzacy) [inline config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml:760]: is this permanent or temporary? consider link the issue for context?

**Review by krzyzacy — APPROVED** [review]

**Pull request merged**